recipes/scala/pom.xml

```diff
@@ -0,0 +1,123 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.adobe.platform.ml</groupId>
+	<artifactId>ml-retail-sample</artifactId>
```

Comment:
Should we have `spark` in the identification somehow?  If not, will this artifactId collide with other samples?  Might be nice to differentiate upon runtime engine.

Reply:
Sure